### DISCUSSION TITLE
Hello world

### DISCUSSION THREAD

**Discussion by Francois-Zinserling** [a-cosmology-group/acg#35] [Q&A] [answered]
Just following Louis's instructions ...

**Comment by Francois-Zinserling** [comment]
Not sure if I am missing some chats/communications. With the emails I used to get on average a few per day, but not sure if it is still happening? Maybe everyone is still settling in?

When I look at the tab 'Discussions' it shows where the topics are debated, and my thinking is this replaces the old email trails. Am I right?

If so, then my difficulty is in seeing which chats/discussions have new comments on them. A NEW topic comes in bold, so easy to see, but when people reply on old topics, there's no indication that there is something new to read. So you scroll to the bottom of the latest discussions, only to see the last comment was 2 weeks ago, or maybe you find a new comment from a few hours ago.

If I'm being a dumkopf (as per usual) please enlighten this poor soul 😸 
Also hoping this comment pops to the top of your feeds so there can be a reply! 😁

**Reply by ftessier** [reply, marked as answer]
I use notifications to get an email for any new discussion activity. From the email side, this looks identical to the email list. When I receive an email notification, I click on the "View on GitHub" link at the bottom, to read the message in the github Discussions interface, which I find more convenient for reading the thread and eventually post a comment or reply.

Email notifications are controlled by the "Watch/Unwatch" button-menu, top right on the repository web page. There you can set the type of traffic for which you want to receive notifications. There is also a setting for notification _type_ (email, etc.) under your own profile settings. Let me know if you need additional directions in this regard.

There is a notification center on github where you can view and manage your notifications: https://github.com/notifications.

For additional details, consult the documentation for notifications, quite interesting (looked at it just now and still learned a few things!):

https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/about-notifications

**Reply by RedshiftDrift** [reply]
@Francois-Zinserling see the note at the bottom of the last message I sent two days ago from [A Cosmology Group]

Did you receive it?
